src/test/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMBuilderTest.java

```diff
@@ -71,17 +71,22 @@ public GitHubSCMBuilderTest(boolean configuredByUrl){
         this.configuredByUrl = configuredByUrl;
     }
 
-    @Before
-    public void setUp() throws IOException {
-        owner = j.createProject(WorkflowMultiBranchProject.class);
+    public void createGitHubSCMSourceForTest(boolean configuredByUrl, String repoUrlToConfigure) throws Exception {
```

Comment:
nit: you could use a signature like this:
```suggestion
    public void createGitHubSCMSourceForTest(String repoUrlToConfigure) throws Exception {
```

And then use `repoUrlToConfigure != null` wherever you use `configuredByUrl` in this method.